+vis/image.m

```diff
@@ -55,34 +62,34 @@
 end
 
 % Add a new subscriptable origin signal to the same network as the input
-% signal, t, and use this to store the stimulus texture layer and
+% signal, 't', and use this to store the stimulus texture layer and
 % parameters
 elem = t.Node.Net.subscriptableOrigin('image');
 elem.azimuth = 0;
 elem.altitude = 0;
-elem.dims = [50,50];
+elem.dims = [50,50]';
 elem.orientation = 0;
 elem.sourceImage = sourceImage;
-elem.colour = [1 1 1];
+elem.colour = [1 1 1]';
 elem.rescale = false;
```

Comment:
You used an outdated version of this function and possibly others in the vis package.  Please merge from dev and check that everything is correctly up to date.

Reply:
Ah, yup ok